lib/conversationEngine.js

```diff
@@ -585,7 +586,15 @@ class ConversationEngine {
     this.topLevelGoals.push(goal);
   };
 
-
+  /**
+   * Override the default launch phrases. Parameter is used directly
+   * with the response.say() function when Alexa launches this app
+   * 
+   * @param {string[]} phrases - response or array of potential responses
+   */
+  setLuanchPhrases(phrases) {
```

Comment:
Typo, should be setLaunchPhrases

Reply:
@muenzpraeger - great catch. I am going to accept the PR and make the changes.